tests/common/snappi/common_helpers.py

```diff
@@ -325,7 +325,7 @@ def pfc_class_enable_vector(prio_list):
     for p in prio_list:
         vector += (2**p)
 
-    return "{:x}".format(vector)
+    return vector
```

Comment:
is this change due to the differences between tgen and snappi?

Reply:
yes, the type is integer now..

![image](https://user-images.githubusercontent.com/48308607/126768857-bc1215a4-6aae-4c5d-81fa-91cfb8f3e3bd.png)